installation/install.sh

```
@@ -4127,6 +4127,7 @@ do
         --sds) SDS_INSTALL='y';shift;;
         --no-zops) SKIP_ZOPS_INSTALL='y';shift;;
         --skip-pjnum) SKIP_PJNUM_CHECK='y';shift;;
+        -C|--precheck-only) PRECHECK_ONLY='y';UPGRADE='y';shift;;
```

Shellcheck (0.11.0)

[warning] This pattern never matches because of a previous pattern on line 4077.

(SC2222)